### PR TITLE
Support additional property sources

### PR DESCRIPTION
**Pull request by aalmiray**
As described in #45 however only explicit properties are supported for the time being.

### CODE REVIEW

File: layrry-config/src/main/java/org/moditect/layrry/config/LayersConfigLoader.java [hunk 1]
```diff
@@ -35,12 +35,28 @@
...
+        return loadConfig(layersConfigFile, new Properties());
+    }
+
+    public static LayersConfig loadConfig(Path layersConfigFile, Path propertiesFile) {
+        Properties properties = new Properties();
+
+        try (InputStream inputStream = propertiesFile.toUri().toURL().openStream()) {
+            properties.load(inputStream);
+        } catch (IOException e) {
+            throw new IllegalArgumentException("Unexpected error reading properties file. " + propertiesFile, e);
```
Comment (gunnarmorling) [inline layrry-config/src/main/java/org/moditect/layrry/config/LayersConfigLoader.java:47]:
```suggestion
            throw new IllegalArgumentException("Unexpected error reading properties file: " + propertiesFile, e);
```

**Review by gunnarmorling — APPROVED** [review]
LGTM, one tiny remark inline. Plus, the README needs updating, too. Thanks!

File: layrry-config/src/main/java/org/moditect/layrry/config/LayersConfigLoader.java (hunk 1 above)
Comment (aalmiray) [reply layrry-config/src/main/java/org/moditect/layrry/config/LayersConfigLoader.java]: Done

**Comment by gunnarmorling** [comment]
Applied; thx!

**Comment by aalmiray** [comment]
This simplifies the JavaFX example once again 😄

**Comment by gunnarmorling** [comment]
Yeah!

**Pull request merged**